AppInspector.CLI/Writers/WriterFactory.cs

```diff
@@ -111,7 +115,37 @@ public static CommandResultsWriter GetExportWriter(CLIExportTagsCmdOptions optio
                     break;
 
                 case "text":
-                    writer = new ExportTextWriter();
+                    writer = new ExportTagsTextWriter();
+                    break;
+
+                default:
+                    WriteOnce.Error(MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_ARG_VALUE, "-f"));
+                    throw new OpException((MsgHelp.FormatString(MsgHelp.ID.CMD_INVALID_ARG_VALUE, "-f")));
+            }
+
+            //assign the stream as a file or console
+            writer.OutputFileName = options.OutputFilePath;
+            writer.TextWriter = GetTextWriter(writer.OutputFileName);
+
+            return writer;
+        }
+
+        private static CommandResultsWriter GetGetTagsWriter(CLIGetTagsCommandOptions options)
+        {
+            CommandResultsWriter? writer;
+
+            switch (options.OutputFileFormat.ToLower())
+            {
+                case "_dummy":
```

Comment:
is this for testing?

Reply:
I don't know what it's used for. I am just following the existing patterns.

Reply:
I looked more into it.  These just throw exceptions. I'm going to get rid of the dummy writers.